src/buildMathML.js

```diff
@@ -471,6 +493,65 @@ groupTypes.underline = function(group, options) {
     return node;
 };
 
+groupTypes.accentunder = function(group, options) {
```

Comment:
Yep.

You know, in MathML, they actually have an `accentunder` tag. I was following that and not the camelCase standard . My bad. I'll change it.